### DISCUSSION TITLE
Request can we add the levelised cost of energy as KPI

### DISCUSSION THREAD

**Discussion by RonaldBerk** [Project-OMOTES/user_feedback#58] [Q&A]
Can we calculate the price or cost per unit of heat delivered based on CAPEX and OPEX over a period of 30 years in €/GJ.
This KPI is useful to decide if the chosen infrastructure is economically feasible.

**Comment by ewoudwerkman** [comment]
That is ofcourse possible, question for me is if this is something the optimizer will calculate this or some other component.  And is it always 30 years, or sometimes another time horizon?

**Comment by jarsarasty** [comment]
Hi @RonaldBerk ,

As discussed in our meeting last week, to account for the different technical life durations of the assets, the levelized cost of heat could be calculated using the discount rate and the technical life of each specific asset. A default value of 30 years could be used for the technical life. Do you have any suggestions for the default discount rate?

Could you please confirm the types of assets for which you would like to see this KPI?

**Comment by ewoudwerkman** [comment]
Assets have an attribute called 'technicalLifetime' (in years), so this could be uses as input for the LCOE calculation. This must be manually filled in by the user for each asset in case a 30 year default lifetime is not correct for that specific asset.

**Reply by jarsarasty** [reply]
Thanks for the clarification, @ewoudwerkman!

**Comment by jarsarasty** [comment]
Hi @RonaldBerk 
Based on this discussion, a [feature request](https://github.com/Project-OMOTES/user_feedback/issues/66) has been created.
